### ISSUE TITLE
In multi select , the label given to multi select is not read by the screen reader.

### ISSUE THREAD

**Issue by Pavithra-Bala** [select2/select2#5982]
**Steps to reproduce :** 

1. Create select that supports multiple selection and add a Label for it.
2. The <Label> tag's "for" value should be given has the "id" of the select tag.
3. Then when we Focus on the Label through tab key, the value of the label has to be read by the screen reader.

**Actual behavior:**

On the  focus of  Label its value  has not been read by the screen reader. Instead it is being focused on the select tag on the entire div is  read by the screen reader.

**Expected behavior:**

On the  focus of  Label its value  has to be read by the screen reader. 

**Libraries**
jQuery version: 2.1.3 (used in the codepens)
Select2 version: 4.0.13

**Environment:**
Google Chrome/Windows/NVDA screen reader.

**Comment by raufozturk** [comment]
<img width="615" alt="Ekran Resmi 2021-02-24 08 18 58" src="https://user-images.githubusercontent.com/15701742/108951606-525ed880-7679-11eb-88ff-08be76244fe1.png">

in my case, the <li> shows me ID instead of TEXT. The <option> tag is ok. How can i fix this?
